js/viz/docs/doccharts.js

```diff
@@ -1853,9 +1853,11 @@ var dxChart = {
      * @extends Action
      * @type function(e)
      * @type_function_param1 e:object
-     * @type_function_param1_field4 axis:chartAxisObject
-     * @type_function_param1_field5 range:VizRange
-     * @type_function_param1_field6 cancel:boolean
+     * @type_function_param1_field4 event:event
+     * @type_function_param1_field5 axis:chartAxisObject
+     * @type_function_param1_field6 range:VizRange
+     * @type_function_param1_field7 cancel:boolean
+     * @type_function_param1_field8 actionType:string
```

Comment:
```suggestion
     * @type_function_param1_field8 actionType:Enums.ChartZoomPanActionType
```

This will require adding a new enum to the docEnums.js file.

Reply:
done